libs/ledger-live-common/src/device/hooks/useLatestFirmware.ts

```diff
@@ -0,0 +1,18 @@
+import { getProviderId } from "../../manager/index";
+import { getEnv } from "@ledgerhq/live-env";
+import { HttpManagerApiRepository } from "../../device-core/managerApi/repositories/HttpManagerApiRepository";
+import { useGetLatestFirmware } from "../../device-react/hooks/useGetLatestFirmware";
+import { DeviceInfoEntity } from "../../device-core/managerApi/entities/DeviceInfoEntity";
+import { HttpManagerApiRepositoryFactory } from "../factories/HttpManagerApiRepositoryFactory";
+
+export function useLatestFirmware(
```

Comment:
[ASK] Why `useLatestFirmware` and not `useGetLatestFirmware` ?

Reply:
we just kept the original name here